### ISSUE TITLE
Parameters for `DfltDisklessStorPool` not showable

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#42]
I can set nic for `DfltDisklessStorPool`:

```
linstor storage-pool set-property $node DfltDisklessStorPool PrefNic data
```

but can't determine is it already set or not.

`linstor -m sp l` and `linstor sp lp $node DfltDisklessStorPool` are both saying nothing about it

**Comment by ghernadi** [comment]
Unfortunately that is correct. Whenever you create a node, it gets automatically a `DfltDisklessStorPool` created. The purpose of this storage pool was to not have to bother creating a diskless storage pool manually in order to deploy a simple diskless resource. That means, if you have a few nodes having actual storage, and many nodes as "diskless only", something simple as `sp l` would bloat the output with all those `DfltDisklessStorPool` entries - that was our thought when we decided to filter the `DfltDisklessStorPool`s out from `sp l`. 

Listing storage pool properties with `sp lp ...` is internally based on the same mechanism as `sp l`, which means that the filtering of `DfltDisklessStorPool` also affects the listing of properties (which explains your reported behavior). 

That said, it was never our intention to allow properties on that storage pool either. I am not sure how we will solve this issue. For now I can come up with three possible solutions:
* List `DfltDisklessStorPool`s if they have some properties set (could be cumbersome, depending on how easy it is to check if there are properties set or not - thinking about default-properties, that might also get detected)
* Skip the filter when explicitly asking for that storage pool
* Do not allow setting properties on `DfltDisklessStorPool`. In this case you can still create (arbitrary many) storage pools with `diskless` type / driver. These manually created diskless storage pools will be visible in both, `sp l` and `sp lp ...`

I will open an internal ticket in discussing how we want to solve this. Until then, you can surely use the suggested method from the last point.

**Comment by kvaps** [comment]
For me diskless pools are fully unusable without specifying network card, as an option we can also show them with some special flag for the cli, eg something like:
```
linstor sp l --show-default-pools
```
But I like an idea for refuse any settings on `DfltDisklessStorPools` also

**Comment by kvaps** [comment]
fixed in v1.4.2
